opm/input/eclipse/Schedule/UDQ/UDQEnums.hpp

```diff
@@ -155,6 +155,7 @@ enum class UDAControl
     WCONPROD_RESV,
     WCONPROD_BHP,
     WCONPROD_THP,
+    WCONPROD_LIFT,
```

Comment:
At some point we'll probably have to integrate that case into the [`keyword`](https://github.com/OPM/opm-common/blob/7938183f231f00493b917ccfa1e99be8961cb8e4/src/opm/input/eclipse/Schedule/UDQ/UDQEnums.cpp#L549) and [`udaCode()`](https://github.com/OPM/opm-common/blob/7938183f231f00493b917ccfa1e99be8961cb8e4/src/opm/input/eclipse/Schedule/UDQ/UDQEnums.cpp#L592) helper functions to enable restart file support.

Reply:
Thanks for pointing to these - had a go at restart support as well now (limited to GRAT ALQ).